packages/variance/src/types/props.ts

```diff
@@ -0,0 +1,46 @@
+export type AbstractProps = Record<string, unknown>;
+
+interface BreakpointKeys<T = string> {
+  xs: T;
+  sm: T;
+  md: T;
+  lg: T;
+  xl: T;
+}
+export interface BreakpointCache {
+  map: BreakpointKeys;
+  array: string[];
+}
+
+export interface AbstractTheme {
+  breakpoints?: BreakpointKeys;
```

Comment:
As we're discussing now:

```suggestion
  breakpoints: BreakpointKeys;
```